modules/ROOT/pages/modify-message-type-settings.adoc

```diff
@@ -0,0 +1,48 @@
+= Modifying Message Types
+
+You can modify a message type's settings, but what you can modify depends on the message type's usage and protocol:
+
+. In the navigation menu, select the *<host-name>* page.
+. Locate the message type you want to modify.
+. Change any of the following settings in the *Edit message type* window:
++
+[%header%autowidth.spread]
+|===
+| Message Type |Format | What You Can Modify
+| Receive from Partners
+| EDIFACT or X12
+a|
+* Select a different schema file, if the *Schema* field is set to *Customize*.
+* Select a different reference map file to map the reference identifiers, business key, and custom attributes.
+* Add new custom message attributes.
+|
```

Comment:
The `Message Type` in this row is empty. If this is intended to be also Receive from Partners, you can use `.2+|` to make a cell span 2 rows, for example: 
.2+| Receive from Partners. 
However, doing so might require to readjust other cells in the table.